frontend/scss/pages/_explorer.scss

```diff
@@ -13,6 +13,7 @@
 
   img {
     width: 100%;
```

Comment:
@andela-iakande + @Sharonsyra do we mean max-width:100% instead?

Reply:
@chegejames Yes, max-width is better used here for scalability purpose. It has been fixed.